src/pages/home/report/ReportActionCompose.js

```diff
@@ -109,6 +109,8 @@ const defaultProps = {
     ...withCurrentUserPersonalDetailsDefaultProps,
 };
 
+const IS_EMPTY_PATTERN = /^(\s|`)*$/;
```

Comment:
We should move this to [CONST.js](https://github.com/Expensify/App/blob/bf03a648133bac2bb834dbff3f230d143ac2aae0/src/CONST.js)